macos/package_files/3.9.3/preinstall.sh

```diff
@@ -23,7 +23,7 @@ if [ `launchctl getenv WAZUH_PKG_UPGRADE` == true ]; then
 fi
 
 if [ `launchctl getenv WAZUH_PKG_UPGRADE` == true ]; then
```

Comment:
Same as before, this notation is legacy and you must change == to =.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/967ac6b13a724cb542685b0664af3c81331bfcca and https://github.com/wazuh/wazuh-packages/pull/221/commits/12ab83e636002b6afad0aa7a71cfc94a5d7bb27e